init/init.go

```diff
@@ -168,10 +171,21 @@ func getLaunchConfig(cfg *config.CloudConfig, dockerCfg *config.DockerConfig) (*
 	return &launchConfig, args
 }
 
+func isInitrd() bool {
+	var stat syscall.Statfs_t
+	syscall.Statfs("/", &stat)
+	return stat.Type == 0x01021994
```

Comment:
This should be checking for both tmps and ramfs.  There is probably a constant somewhere for this, but if there isn't one can you declare a constant and avoid the magic number.  Just makes it obscure what you're doing here.